src/content/blog/post-mortem-esm-sh.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: "Post-mortem: esm.sh Breaking Changes"
+description: Some vals experienced lockfile and React errors due to breaking changes in esm.sh
+pubDate: Jan 16, 2025
+author: Steve Krouse
+---
+
+import { Image } from "astro:assets";
+import Chart from "./post-mortem-esm-sh/chart.svg";
+
+Yesterday [esm.sh](https://esm.sh) deployed breaking changes that caused lockfile errors and client-side React errors for 2% of Val Town users. The issue was resolved by esm.sh rolling back the change related to React version pinning and us manually regenerating our users' affected lockfiles. We are working to upgrade our systems, and with the esm.sh maintainer to ensure this doesn't happen again.
```

Comment:
this is fixing two issues:
- they rolled back a change that broke certain urls
- we also had to invalidate lockfiles because the redirects and content changes

not sure if you wanted to spell that out.